latch_cli/main.py

```diff
@@ -271,12 +271,37 @@ def pad_styled(x: str, l: int, align_right=False):
     about local development.
     """,
 )
-@click.argument("pkg_root", nargs=1, type=click.Path(exists=True))
-def local_execute(pkg_root: Path):
+@click.argument("pkg_root", nargs=1, type=click.Path(exists=True))@click.option(
+    "-u",
+    "--use-auto-version",
+    is_flag=True,
+    default=False,
+    type=bool,
+    help=(
+        "Whether to automatically bump the version of the workflow each time register"
+        " is called. This should only be used if you update the Dockerfile, but don't"
+        " want to permanently register those changes to Latch."
+    ),
+)
+@click.option(
+    "-o",
+    "--output-dir",
+    default="outputs",
+    type=str,
+    help=(
+        "The directory in which the outputs will be written to by the workflow. "
+        "(/root/{output_dir}). Defaults to 'outputs'."
```

Comment:
id say make `output_dir = '/root/outputs'` directly instead of prepending `/root` (e.g. things won't work as expected if someone does `latch local-execute -u --output-dir='/root/outputs'` instead of `latch local-execute -u --output-dir='outputs'`)